react/store/ReturnRequest/StoreReturnList.tsx

```diff
@@ -1,37 +1,48 @@
 import React from 'react'
-import { Link } from 'vtex.render-runtime'
-import { Button } from 'vtex.styleguide'
+import { useRuntime } from 'vtex.render-runtime'
+import { Button, PageBlock } from 'vtex.styleguide'
+import { ContentWrapper } from 'vtex.my-account-commons'
+import { FormattedMessage } from 'react-intl'
 
+import ListTable from '../../common/components/returnList/ListTable'
 import { useReturnRequestList } from '../../hooks/useReturnRequestList'
 
 export const StoreReturnList = () => {
+  const { navigate } = useRuntime()
+
   const { returnRequestData } = useReturnRequestList()
-  const { data, loading, error } = returnRequestData
-  const { returnRequestList } = data ?? {}
-  const { list, paging } = returnRequestList ?? {}
+  const { loading } = returnRequestData
+
+  const handleNewRequest = () => {
+    navigate({ to: '#/my-returns/add' })
+  }
 
-  // eslint-disable-next-line no-console
-  console.log({ error })
+  const headerContent = (
+    <Button
+      variation="primary"
+      size="small"
+      isLoading={loading}
+      onClick={() => handleNewRequest()}
```

Comment:
Making this button an anchor tag solves the issue of the button being clickable only once.
 
```suggestion
      href="#/my-returns/add"
```